android/core/src/main/java/com/stadiamaps/ferrostar/core/NavigationViewModel.kt

```diff
@@ -85,7 +88,7 @@ class DefaultNavigationViewModel(
               started = SharingStarted.WhileSubscribed(),
               initialValue =
                   uiState(
-                      ferrostarCore.state.value, spokenInstructionObserver?.isMuted, lastLocation))
+                      ferrostarCore.state.value, spokenInstructionObserver?.isMuted, locationProvider.lastLocation, locationProvider.lastLocation))
```

Comment:
Do we want to use the last location for both cases here? I think should be the unsnapped value and the other could be snapped.